NebulaHost/PacketProcessors/Factory/Foundation/FoundationBuildUpdateProcessor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.DataStructures;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Factory;
+using NebulaModel.Packets.Processors;
+using NebulaWorld.Factory;
+using UnityEngine;
+
+namespace NebulaHost.PacketProcessors.Factory.Foundation
+{
+    [RegisterPacketProcessor]
+    class FoundationBuildUpdateProcessor : IPacketProcessor<FoundationBuildUpdatePacket>
```

Comment:
Are these two files identical? Why not use linked files?

Reply:
I think there are more cases where PacketProcessor files are completely the same on the client and on the host. I guess we can use linked files, I just have to check, how it is done.
Also, I think the namespace needs to be:
for host - namespace NebulaHost.*
for client- namespace NebulaClient.*

I guess. But that is good note.

Reply:
There is a discussion on Discord about this, @hubastard wants to think on it first.